src/chart/components/chart/chart.component.ts

```diff
@@ -154,6 +161,17 @@ export class ChartComponent extends ChartBaseElement {
 				this.dynamicObjects.model.removeObject(series.id);
 			}),
 		);
+		// redraw background only when chart is resized
+		this.addRxSubscription(
+			this.canvasBoundsContainer.observeAnyBoundsChanged().subscribe(() => {
+				this.backgroundDrawPredicateSubj.next(false);
+			}),
+		);
+		this.addRxSubscription(
+			this.chartResizeHandler.canvasResized.subscribe(() => {
+				this.backgroundDrawPredicateSubj.next(true);
```

Comment:
we use full name of Subject, there are no places with short version: `backgroundDrawPredicateSubject`